packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -176,6 +176,17 @@ granting permissions to other AWS accounts or organizations.
 
 [Example of Lambda Layer usage](test/integ.layer-version.lit.ts)
 
+### Layer Versioning and Removal Policies
+
+By default, updating a layer creates a new version and deletes the current version managed by the stack. Alternatively, a removal policy can be used to retain the current version:
```

Comment:
Something like this and I don't think you need a separate section for this - 

```suggestion
By default, updating a layer creates a new layer version, and CloudFormation will delete the old version as part of the stack update.
Alternatively, a removal policy can be used to retain the current version:
```

Reply:
that sounds much clearer, thanks